timesketch/api/v1/resources/archive.py

```diff
@@ -481,39 +628,153 @@ def _archive_sketch(self, sketch: Sketch):
         for label in labels_to_prevent_deletion:
             if sketch.has_label(label):
                 abort(
-                    HTTP_STATUS_CODE_FORBIDDEN,
-                    "A sketch with the label {:s} cannot be " "archived.".format(label),
+                    HTTP_STATUS_CODE_BAD_REQUEST,
```

Comment:
Is this really a BAD REQUEST if the sketch has a label that prevents archival? Wouldn't the old FORBIDDEN be a better match?